aiidalab_widgets_base/structures.py

```diff
@@ -114,68 +125,84 @@ def store_structure(self, label=None, description=None):
         if self.structure_node is None:
             return
         if self.structure_node.is_stored:
-            print("Already stored in AiiDA: " + repr(self.structure_node) + " skipping..")
+            self.output.value = "Already stored in AiiDA [{}], skipping...".format(self.structure_node)
             return
         if label:
             self.structure_node.label = label
         if description:
             self.structure_node.description = description
         self.structure_node.store()
-        print("Stored in AiiDA: " + repr(self.structure_node))
+        self.output.value = "Stored in AiiDA [{}]".format(self.structure_node)
 
-    @observe('node_class')
-    def _change_structure_node(self, change):
-        if change['new'] is not None:
-            self.structure = self._inserted_structure
+    @default('node_class')
+    def _default_node_class(self):
+        return 'StructureData'
```

Comment:
Is it really necessary to support multiple node classes?

Reply:
I don't see a reason why not doing this: if there are different structure types in AiiDA, we should support them.

Reply:
Well, I'm asking because whenever we introduce something like this all downstream code must perform checks about the data type they receive and then change behavior. So that would be a reason "why not".

Reply:
I think the reason for having it possible to select the output structure type is exactly to prevent downstream code of doing any conversions. The widget allows setting up the output node type prior to its usage.

To give a concrete example: "a downstream code" can be a workflow. When setting up a workflow user knows already which input structure type is required there (say `CifData`). So using the current functionality an app developer can already select `structure_node` of which type will be produced by the widget.